examples/vnet/doca_flow.h

```diff
@@ -193,7 +193,8 @@ struct doca_flow_fwd {
         } rss;
 
         struct port {
-                int id;
+                uint16_t id;
+                bool hairpin;
```

Comment:
hairpin is not part of doca_flow
lets discuss

Reply:
hairpin is just a bool that we set upon building the fwd.
setting this bool as true indicates that we are in nic mode then the "id" is the hairpin's queue index.
setting this bool as false indicates that we are in switchdev.

this is done in the application. can be changed i think